@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -230,7 +238,7 @@
 /* dropdown & non selectable dropdown items */
 
 .navds-combobox__list {
-  max-height: 290px;
```

Comment:
This size is a little arbitrary, but helps a little with this issue: https://github.com/navikt/aksel/issues/3498. By adjusting the height, you will by default see part of the last item before scrolling.